lib/module.js

```diff
@@ -71,10 +66,10 @@ function readPackage(requestPath) {
     return packageMainCache[requestPath];
   }
 
-  try {
-    var jsonPath = path.resolve(requestPath, 'package.json');
-    var json = fs.readFileSync(jsonPath, 'utf8');
-  } catch (e) {
+  var jsonPath = path.resolve(requestPath, 'package.json');
+  var json = internalModuleReadFile(jsonPath);
```

Comment:
I don't really disagree but the other two call sites don't seem to be bottlenecks like the package.json load step is.